src/lxml/tests/test_isoschematron.py

```diff
@@ -269,15 +269,15 @@ def test_schematron_result_report(self):
         self.assertTrue(
             isinstance(schematron.validation_report, etree._ElementTree),
             'expected a validation report result tree, got: %s' %
-            (schematron.validation_report))
+            schematron.validation_report)
 
         schematron = isoschematron.Schematron(schema, store_report=False)
         self.assertTrue(schematron(tree_valid), schematron.error_log)
         valid = schematron(tree_invalid)
         self.assertTrue(not valid)
         self.assertTrue(schematron.validation_report is None,
             'validation reporting switched off, still: %s' %
-            (schematron.validation_report))
+                        schematron.validation_report)
```

Comment:
Since the indentation is inconsistent between the two changes now, I'd prefer having both at the and of the previous line, right after the `%`.